Open-Source-Pundits/nutshell
# Ts testing

# Description

fixed session issue with POST

Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. `git fetch --all`
2 `git checkout ts-testing`
3. `git pull origin ts-testing`
4. Make sure you can create a new session with a new tab, create an account, create an event, and it will POST to DOM


# Checklist:
